app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -393,6 +393,9 @@ def process_enrollment_response(enrollment, response)
     else
       handle_unsupported_status(enrollment, response)
     end
+
+    # invoke job to send sms notification
+    send_enrollment_status_sms_notification(enrollment: enrollment)
```

Comment:
You will need to update or add a new call to this method in order to properly remove phone numbers for expired enrollments, in addition to the other change you made.

Reply:
I'm not remembering a message being sent for expired enrollments. If we don't send a message, then that could also be a lifecycle callback instead of an update to this function call.

Reply:
@NavaTim , id does send message for expired in handle_bad_request_error method, controlled by in_person_stop_expiring_enrollments flag, and it set status as expired as the same time, otherwise it set it as incomplete.

Reply:
@NavaTim @gina-yamada all comments addressed.

Reply:
Deleting the phone number upon cancellation was not resolved by the changes.

Reply:
Apologies for missing this conversation. [The deletion is being worked on in my branch here](https://github.com/18F/identity-idp/compare/main...brittany/lg-10167-delete-phone-no-when-enroll-cancelled).